src/fides/lib/oauth/roles.py

```diff
@@ -67,14 +77,24 @@ class RoleRegistry(Enum):
     CONNECTION_READ,
     CONSENT_READ,
     CONNECTION_TYPE_READ,
+    CTL_DATASET_READ,
+    DATA_CATEGORY_READ,
+    CTL_POLICY_READ,
+    DATA_QUALIFIER_READ,
     DATAMAP_READ,
     DATASET_READ,
+    DATA_SUBJECT_READ,
+    DATA_USE_READ,
+    EVALUATION_READ,
+    ORGANIZATION_READ,
     POLICY_READ,
     PRIVACY_REQUEST_READ,
     PRIVACY_REQUEST_NOTIFICATIONS_READ,
+    REGISTRY_READ,
     RULE_READ,
     SCOPE_READ,
     STORAGE_READ,
+    SYSTEM_READ,
```

Comment:
Adding newly-created "read" scopes to the Viewer role, but not that the roles->scopes mapping will be overhauled here https://github.com/ethyca/fides/issues/2720